framework_tvb/tvb/adapters/analyzers/wavelet_adapter.py

```diff
@@ -62,16 +62,37 @@ class WaveletAdapterModel(ViewModel, ContinuousWaveletTransform):
         doc="""The timeseries to which the wavelet is to be applied."""
     )
 
+    f_lo = Attr(
+        field_type=float,
+        required=True,
+        label='Lo',
+        default=ContinuousWaveletTransform.frequencies.default.lo,
+        doc='start of range'
+    )
+
+    f_hi = Attr(
+        field_type=float,
+        required=True,
+        label='Hi',
+        default=ContinuousWaveletTransform.frequencies.default.hi,
+        doc='end of range'
+    )
+
+    f_step = Attr(
+        field_type=float,
+        required=True,
+        label='Step',
+        default=ContinuousWaveletTransform.frequencies.default.step,
+        doc='step of range'
+    )
+
 
 class RangeForm(Form):
     def __init__(self, prefix=''):
         super(RangeForm, self).__init__(prefix)
-        self.lo = SimpleFloatField(self, name='lo', required=True, label='Lo', doc='start of range')
-        # default=ContinuousWaveletTransform.frequencies.lo)
-        self.step = SimpleFloatField(self, name='step', required=True, label='Step', doc='step of range')
-        # default=ContinuousWaveletTransform.frequencies.step)
-        self.hi = SimpleFloatField(self, name='hi', required=True, label='Hi', doc='end of range')
-        # default=ContinuousWaveletTransform.frequencies.hi)
+        self.lo = FloatField(WaveletAdapterModel.f_lo, self, name='Lo')
+        self.hi = FloatField(WaveletAdapterModel.f_hi, self, name='Hi')
+        self.step = FloatField(WaveletAdapterModel.f_step, self, name='Step')
```

Comment:
So, related to the comment above, put directly here:
self.lo = FloatField(Float(label='Lo', default=ContinuousWaveletTransform.frequencies.default.lo, doc='start of range'), self, name='Lo')